src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -315,6 +311,16 @@ def count_valid(self) -> int:
         """
         return self._buffer.count_valid()
 
+    @property
+    def count_covered(self) -> int:
```

Comment:
Same as with the ring buffer, just for the records, feel free to close here to keep the discussion in one place.